src/muxy.ts

```diff
@@ -520,6 +530,10 @@ export class Muxy implements MuxyInterface {
       Muxy.printInfo();
     }
 
+    if (options.transactionsEnabled) {
+      this.transaction.onUserPurchase(() => {});  
```

Comment:
This callback should send the message to Muxy's backend. And that call should be removed from the default implementation.

Reply:
So where is this getting the tx from (that's used in that call in the default implementation)

Reply:
```this.transaction.onUserPurchase((tx: GenericTransaction) => {
  // sdk.dothething()
})```